src/Model/Block.php

```diff
@@ -61,4 +61,21 @@ public function addNode(Object $node): Block
         $this->nodes[] = $node;
         return $this;
     }
+
+    public function getText(): string
+    {
+        $text = '';
+        foreach ($this->nodes as $node) {
+            if ($node instanceof Text) {
+                $text .= $node->getText();
+
+            } elseif ($node instanceof Node) {
+                $text .= $node->getText();
+
+            } else {
+                throw new \LogicException('WTF! This should never happen');
```

Comment:
Let's improve the error message please. _"WTF! This should never happen"_ is neither helpful nor really nice / professional.

Reply:
You're right.